content/docs/5.1/deploy/operate/maintain.md

```diff
@@ -0,0 +1,206 @@
+---
+title: TiUP Common Operations
+description: Learn the common operations to operate and maintain a TiKV cluster using TiUP
+menu:
+    "5.1":
+        parent: Operate TiKV
+        weight: 3
+---
+
+This document describes the following common operations when you operate and maintain a TiKV cluster using TiUP.
+
+- [View the cluster list](./#view-the-cluster-list)
+- [Start the cluster](./#start-the-cluster)
+- [View the cluster status](./#view-the-cluster-status)
+- [Modify the configuration](./#modify-the-configuration)
+- [Rename the cluster](./#rename-the-cluster)
+- [Stop the cluster](./#stop-the-cluster)
+- [Clean up cluster data](./#clean-up-cluster-data)
+- [Destroy the cluster](./#destroy-the-cluster)
+
+## View the cluster list
+
+You can manage multiple TiKV clusters using the TiUP cluster component. When a TiKV cluster is deployed, the cluster appears in the TiUP cluster list.
+
+To view the list, run the following command:
+
+```shell
+tiup cluster list
+```
+
+## Start the cluster
+
+The components in the TiKV cluster are started in the following order (The monitoring component is also started):
+
+**PD -> TiKV**
+
+To start the cluster, run the following command:
+
+```shell
+tiup cluster start ${cluster-name}
+```
+
+You can start only some of the components by adding the `-R` or `-N` parameters in the command. For example:
+
+- This command starts only the PD component:
+
+    ```shell
+    tiup cluster start ${cluster-name} -R pd
+    ```
+
+- This command starts only the PD components on the `1.2.3.4` and `1.2.3.5` hosts:
+
+    ```shell
+    tiup cluster start ${cluster-name} -N 1.2.3.4:2379,1.2.3.5:2379
+    ```
+
+{{< info >}}
+If you start the specified component by using the `-R` or `-N` parameters, make sure the starting order is correct. For example, start the PD component before the TiKV component. Otherwise, the start might fail.
```

Comment:
```suggestion
If you start the components with `-R` or `-N` parameters, make sure the order of components is correct. For example, start the PD component before the TiKV component. Otherwise, the start might fail.
```